pcl_ros/include/sixriver/gpu/filters/impl/voxel_grid.hpp

```diff
@@ -311,10 +318,16 @@ sixriver::VoxelGrid<PointT>::applyFilter (PointCloud &output)
             int idx = ijk0 * divb_mul_[0] + ijk1 * divb_mul_[1] + ijk2 * divb_mul_[2];
             index_vector.push_back (cloud_point_index_idx (static_cast<unsigned int> (idx), *it));
         }
+        if (detect_negative_points_ && (number_of_negative_points > negative_point_number_threshold_))
+        {
+            ROS_WARN_THROTTLE(3.0, "Detecting %u point(s) below height threshold", number_of_negative_points);
```

Comment:
is there a reason why we are throttling data here?

Reply:
The reason was to avoid spamming log, but I can remove that though